From dc0b4e0aaba3b4fac1827b9d4879db7cf2a6224f Mon Sep 17 00:00:00 2001
From: Xavier Boniface <xavierx.boniface@intel.com>
Date: Thu, 2 May 2013 16:25:31 +0200
Subject: [PORT FROM MAIN][BT] Add bdt to userdebug builds

BZ: 104787

Needed by PPD because Bluetooth RF certification tests must be ran on
userdebug builds.

Note: you need to do an 'adb root' prior to run bdt on userdebug builds.
Otherwise, you will encounter some perimission issues.

Change-Id: I1f8423565689f97d9816e10dcf68b189271cb0cc
Orig-Change-Id: Ie14007e119f50ac4b95a4f17e74d58e2a5df274b
Feature: BT
Category: Improvement
Purpose: AOSP+
Origin: Internal
Upstream-candidate: no
Signed-off-by: Xavier Boniface <xavierx.boniface@intel.com>
Reviewed-on: http://android.intel.com:8080/109275
Reviewed-by: cactus <cactus@intel.com>
Reviewed-by: Bondier, CedricX <cedricx.bondier@intel.com>
Tested-by: Zurmely, PierreX <pierrex.zurmely@intel.com>
Reviewed-by: buildbot <buildbot@intel.com>
Tested-by: buildbot <buildbot@intel.com>
---
 test/bluedroidtest/Android.mk |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/test/bluedroidtest/Android.mk b/test/bluedroidtest/Android.mk
index 0c6693a..9f79f26 100644
--- a/test/bluedroidtest/Android.mk
+++ b/test/bluedroidtest/Android.mk
@@ -23,7 +23,7 @@ LOCAL_SRC_FILES:=     \
 
 LOCAL_C_INCLUDES :=
 
-LOCAL_MODULE_TAGS := eng
+LOCAL_MODULE_TAGS := eng debug
 
 LOCAL_MODULE:= bdt
 
-- 
1.7.4.1

